doctr/export_as_pdfa.ipynb

```diff
@@ -28,6 +28,8 @@
     "!pip3 install reportlab>=3.6.2\n",
     "# Optional if you want to merge multiple pdfs\n",
     "!pip3 install PyPDF2==1.26.0\n",
+    "# For matplotlib\n",
```

Comment:
Instead of this we can change

`!pip3 install python-doctr[tf]@git+https://github.com/mindee/doctr.git`
to
`!pip3 install python-doctr[tf,viz]@git+https://github.com/mindee/doctr.git`